tools/cnvkit-batch.cwl

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: "cwl:draft-3"
+
+class: CommandLineTool
+baseCommand: ['cnvkit.py', 'batch']
+
+requirements:
+  - class: InlineJavascriptRequirement
+
+description: |
+  Run the complete CNVkit pipeline on one or more BAM files.
+
+inputs:
+
+
+- id: bam_files
+  type:
+  - "null"
+  - type: array
+    items: string
+
+  description: Mapped sequence reads (.bam)
+  inputBinding:
+    position: 1
+
+
+- id: male_reference
+  type: ["null", boolean]
+  default: null
+  description: Use or assume a male reference (i.e. female samples will have +1
+                log-CNR of chrX; otherwise male samples would have -1 chrX).
+  inputBinding:
+    position: 2
+    prefix: --male-reference 
```

Comment:
CWL tip: for Argparse only the position dependent arguments need their position specified. Arguments that have a prefix like `--male-reference` can occur in any order, so it would be nice if `cwlargparse` didn't specify the unneeded positions in these cases.